fluent-bit-plugin: Auto add Kubernetes labels to Loki labels (#1204)
grafana/loki: cmd/fluent-bit/README.md

| Labels        | labels for API requests.                       | {job="fluent-bit"}                    |
| LogLevel      | LogLevel for plugin logger.                    | "info"                              |
| RemoveKeys    | Specify removing keys.                         | none                                |

> | AutoKubernetesLabels | If set to ture, it will auto parse kubernetes metadata into labels | false    |

Reviewer: ```suggestion
| AutoKubernetesLabels | If set to true, it will add all kubernetes labels to loki labels | false    |
```
